### PR TITLE
Sviatlana's branch new

### PR DESCRIPTION
**Pull request by SvitlanaRamanauskas**
Please, review changes from my branch

### CODE REVIEW

File: Controllers/LocationController.cs [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        }
+        [HttpDelete("{id}")]
+        public void DeleteLocation(string id)
+        {
+            LocationRepository.DeleteLocationById(id);
+        }
+        [HttpPut("{id}")]
+        public Location? UpdateLocation(string id, [FromBody] Location location)
+        {
+            if(location.Latitude == null)
```
Comment (Libret) [inline Controllers/LocationController.cs:45]: Latitude and longitude can`t be null

File: Models/Errand.cs [hunk 2]
```diff
@@ -5,24 +5,25 @@ namespace Geonote.Models
...
-        
+
         public Location? Location { get; set; }
-        
 
-        //add optional Address?
-        //add optional Place?
+        public Address? Address { get; set; }
+
+        public List<Place>? PlacesList { get; set; }
```
Comment (Libret) [inline Models/Errand.cs:21]: We need Place like list, but how to made it work?

File: Models/Location.cs [hunk 3]
```diff
@@ -8,10 +8,21 @@ public class Location
 
         public string Longitude { get; set; }
 
-        public Place Place { get; set; }
+        public List <Place> Place { get; set; }
```
Comment (Libret) [inline Models/Location.cs:11]: its ok, we also need Place like list here.

File: Models/Location.cs [hunk 4]
```diff
@@ -8,10 +8,21 @@ public class Location
...
 
+        public Location(string id)
```
Comment (Libret) [inline Models/Location.cs:13]: we dont have empty cinstructor, i prefer add cleen-empty constructor

File: Models/Location.cs [hunk 5]
```diff
@@ -8,10 +8,21 @@ public class Location
...
+        {
+            this.Id = id;
```
Comment (Libret) [inline Models/Location.cs:15]: project doesnt need constructor with ID cose we create it when use AddNewObj()

File: Program.cs [hunk 6]
```diff
@@ -32,27 +32,48 @@
 //Geonote.Repositories.Migrations.Run();
 //var sqliteConnection = Geonote.Repositories.SQLiteConnect.CreateConnection();
 //Geonote.Repositories.Migrations.run(sqliteConnection);
```
Comment (Libret) [inline Program.cs:34]: maybe we need to remove all coments in Programm.cs

File: Repositories/CategoryRepository.cs [hunk 7]
```diff
@@ -10,8 +12,8 @@ public class CategoryRepository
 
         public static void AddNewCategory(string categoryName)
         {
-            var id = Guid.NewGuid();
```
Comment (Libret) [inline Repositories/CategoryRepository.cs:13]: if else for ID

File: Repositories/ErrandRepository.cs [hunk 8]
```diff
@@ -81,13 +85,13 @@ public static List<Errand> GetAllErrands()
                 "LEFT JOIN Category ON Errand.CategoryId = Category.Id\n" +
                 "LEFT JOIN Item ON Errand.Id = Item.ErrandId\n" +
                 "LEFT JOIN Location ON Errand.LocationId = Location.Id\n" +
-                "LEFT JOIN Place ON Errand.PlaceId = Place.Id\n" +
```
Comment (Libret) [inline Repositories/ErrandRepository.cs:84]: why did you remove Place?
Can you rewrite Place place like List<Place> place

File: Repositories/Migrations.cs [hunk 9]
```diff
@@ -12,10 +12,10 @@ public Migrations()
 
         public static void Run() // TA: Create a checkup if this method has been already run to avoid trying to make duplicates and crushing the app
         {
-            SQLTableManagement.CreateTable("CREATE TABLE Errand (Id VARCHAR(20) NOT NULL, Name VARCHAR(200) NOT NULL, Comment VARCHAR(200), CategoryId VARCHAR(20), LocationId VARCHAR(20), PlaceId VARCHAR(20), PRIMARY KEY(Id))");
```
Comment (Libret) [inline Repositories/Migrations.cs:15]: errand has Place like list

File: Repositories/Migrations.cs [hunk 10]
```diff
@@ -12,10 +12,10 @@ public Migrations()
...
+            SQLTableManagement.CreateTable("CREATE TABLE Errand (Id VARCHAR(20) NOT NULL, Name VARCHAR(200) NOT NULL, Comment VARCHAR(200), CategoryId VARCHAR(20), LocationId VARCHAR(20), AddressId VARCHAR(20), PRIMARY KEY(Id))");
             SQLTableManagement.CreateTable("CREATE TABLE Category (Id VARCHAR(20) NOT NULL, Name VARCHAR(200) NOT NULL, PRIMARY KEY(Id))");
             SQLTableManagement.CreateTable("CREATE TABLE Item (Id VARCHAR(20) NOT NULL, Name VARCHAR(200) NOT NULL, ErrandId VARCHAR(20), PRIMARY KEY(Id))");
-            SQLTableManagement.CreateTable("CREATE TABLE Location (Id VARCHAR(20) NOT NULL, Longitude VARCHAR(200) NOT NULL, Latitude VARCHAR(200) NOT NULL, PlaceId VARCHAR(20), PRIMARY KEY(Id))");
```
Comment (Libret) [inline Repositories/Migrations.cs:18]: errand must have architecture of place at least somwhere

File: Controllers/LocationController.cs (hunk 1 above)
Comment (SvitlanaRamanauskas) [reply Controllers/LocationController.cs:45]: In this case I update latitude and longitude separately, by location id. Firstly Latitude, then Longitude. I tried to find method to update them simultaneously, but failed.
